crates/proof-of-sql/src/base/database/column_type_operation.rs

```diff
@@ -207,6 +207,36 @@ pub fn try_cast_types(from: ColumnType, to: ColumnType) -> ColumnOperationResult
     }
 }
 
+/// Verifies that `from` can be cast to `to`. This only supports converting types to a larger type (Decimal(10, 5) to Decimal (11, 5) for example).
```

Comment:
Please explain what "larger type" means so that any external user or future contributor understands it